src/public/interfaces.ts

```diff
@@ -322,6 +322,12 @@ export interface Context {
    * Represents whether PSTN calling is allowed for the current logged in User
    */
   isPSTNCallingAllowed?: boolean;
+
+  /**
+   * The OneNote section ID that is linked to the channel.
+   * Used in Class/Staff/PLC teams.
```

Comment:
NIT: You can remove the "Used in Class/Staff/PLC teams." part of the comment since this property will actually be set even in old channels that aren't part of Class/Staff/PLC teams.